CODEOWNERS

```diff
@@ -132,6 +134,7 @@
 /gallery/communicationservices-insights/** @microsoft/acs-data
 /gallery/workbook/Microsoft.Communication-CommunicationServices.json @microsoft/acs-data
 /gallery/laws-insights/** @microsoft/loganalytics
+/gallery/update-center-insights/** @microsoft/azure-update-center
```

Comment:
is this for update complicance?  it looks like the workbook type that got added in the code side of insights is this:

```
  updateComplianceInsights: "updatecompliance-insights",
```

can you verify this with @yves-chan ?

Reply:
It's unrelated

We want to contribute to azure workbooks for our service – Azure Update Management Center.

Team (Microsoft organization) : [azure-update-center · Microsoft Discussions (github.com)](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Forgs%2Fmicrosoft%2Fteams%2Fazure-update-center&data=05%7C01%7CSwapnil.Jain%40microsoft.com%7C32fcc386914048b7e93208da64db4038%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637933190140192460%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=eh8rAo1YRGj1rCWMhI5HBCRok60VMYtBUWiLWN6jR2w%3D&reserved=0)
Service: [Update management center (Preview) - Microsoft Azure](https://ms.portal.azure.com/#view/Microsoft_Azure_Automation/UpdateCenterMenuBlade/~/overview)
Workbook (that we want to integrate): [3d4242b5-32e7-4b9a-9504-1f7e2da2201f (AUMV2Mirrors) - Microsoft Azure](https://ms.portal.azure.com/#@microsoft.onmicrosoft.com/resource/subscriptions/783fd652-64f3-4680-81e9-0b978c542005/resourceGroups/swjai-global-rg/providers/microsoft.insights/workbooks/3d4242b5-32e7-4b9a-9504-1f7e2da2201f/workbook)

Brief about our service: 
You can use update management center in Azure to centrally manage operating system updates, update configuration settings, and manage the process of installing required updates for your Windows and Linux virtual machines (VMs) in Azure, physical or VMs in on-premises environments, and in other cloud environments. You can quickly assess the status of available updates and manage the process of installing required updates for your machines reporting to update management center.